api/envoy/api/v2/lds.proto

```diff
@@ -54,16 +54,6 @@ message Listener {
   // :ref:`FilterChain <envoy_api_msg_listener.FilterChain>` with the most specific
   // :ref:`FilterChainMatch <envoy_api_msg_listener.FilterChainMatch>` criteria is used on a
   // connection.
-  //
-  // .. attention::
-  //
-  //   In the current version, multiple filter chains are supported **only** so that SNI can be
-  //   configured. See the :ref:`FAQ entry <faq_how_to_setup_sni>` on how to configure SNI for more
```

Comment:
Can we still link to the FAQ from somewhere relevant (or multiple places)? This is a super common question and it's still going to confuse people so would like to maximize cross linking.

Reply:
Re-added.